qa/tasks/cephfs/test_forward_scrub.py

```diff
@@ -232,7 +232,8 @@ def test_inotable_sync(self):
         self.mount_a.umount_wait()
 
         with self.assert_cluster_log("inode table repaired", invert_match=True):
-            self.fs.mds_asok(["scrub_path", "/", "repair", "recursive"])
+            out_json = self.fs.mds_asok(["scrub_path", "/", "repair", "recursive"])
+            self.assertEqual(out_json, {})
```

Comment:
When we eventually fill in this JSON object with useful output, the test will unnecessarily break. I think it's enough to confirm `out_json is not None`.